cumulus_etl/deid/scrubber.py

```diff
@@ -328,7 +328,7 @@ def _check_extensions(
                 "http://hl7.org/fhir/StructureDefinition/geolocation",
                 "http://hl7.org/fhir/StructureDefinition/iso21090-TEL-address",
                 "http://hl7.org/fhir/StructureDefinition/patient-birthPlace",
-                "http://hl7.org/fhir/StructureDefinition/patient-mothersMaidenName"
+                "http://hl7.org/fhir/StructureDefinition/patient-mothersMaidenName",
```

Comment:
I looked at ruff config options for this - and they (the ISC ones) conflict with the formatter. It seems that implicit concatenation across lines like this is how the style wants such lines.

So it's not easy to safeguard against this in the future via ruff check... Just gotta be careful I guess.